### PR TITLE
Gitpodify jenkins design library plugin

### PR DESCRIPTION
**Pull request by Nancy-Chauhan**
This PR adds option to open this project in [Gitpod](https://github.com/gitpod-io) 

[Open in Gitpod](https://gitpod.io/#https://github.com/jenkinsci/design-library-plugin)

### CODE REVIEW

**Review by oleg-nenashev — APPROVED** [review]
Works on my ~machine~ browser

**Comment by oleg-nenashev** [comment]
Ideally we should optimize downloading Maven cache though

**Comment by timja** [comment]
> Ideally we should optimize downloading Maven cache though

There’s settings in gitpod projects you can do for that, enable pre-build and reuse previous one i think. Not sure if you can do that via config file

**Comment by Nancy-Chauhan** [comment]
> > Ideally we should optimize downloading Maven cache though
> 
> There’s settings in gitpod projects you can do for that, enable pre-build and reuse previous one i think. Not sure if you can do that via config file

Hey @timja , prebuilds runs by default (https://www.gitpod.io/docs/prebuilds). 
Additionally Gitpod sets maven settings to cache maven dependencies

File: .gitpod.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+tasks:
+  - init: npm install && npm run build && mvn install -DskipTests=false
```
Comment (timja) [inline .gitpod.yml:2]:
```suggestion
  - init: mvn install -P quick-build
```

File: README.md [hunk 2]
```diff
@@ -14,6 +14,13 @@ This plugin contains:
 
 ## Usage
 
-Run `mvn hpi:run`
+You can open this project as a [Gitpod workspace](https://www.gitpod.io/) which comes pre-configured with all the tools you will need.
+
+[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/jenkinsci/design-library-plugin)
+
+If you prefer using IntelliJ IDEA, you can setup Gitpod integration with JetBrains Gateway using the instructions given [here](https://www.gitpod.io/docs/ides-and-editors/intellij),
```
Comment (timja) [inline README.md:21]:
```suggestion
If you prefer using IntelliJ IDEA, you can setup Gitpod integration with JetBrains Gateway using the instructions on [gitpod.io](https://www.gitpod.io/docs/ides-and-editors/intellij), 
```

**Review by timja — APPROVED** [review]
Let's try it

**Comment by timja** [comment]
🎉 

image

**Comment by timja** [comment]
Hmm I just get redirected to http://localhost:8080 after opening that so it's not working

**Comment by timja** [comment]
https://github.com/jenkinsci/design-library-plugin/pull/63

**Pull request merged**